test/controllers/tracks/exercises_controller_test.rb

```diff
@@ -12,7 +12,7 @@ class Tracks::ExercisesControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "index: renders correctly for joined" do
-    user = create :user
+    user = create :user, :onboarded
```

Comment:
I think we should presume that all users are onboarded and set these to be default in the factory. Then if we **dont'** want someone to be onboarded (the edge-case) have a trait for `:not_onboarded`. 

It's worth having done it this way to check everything is right first, but if you could invert it now, I'd appreciate it 💙